cli/config/config.go

```diff
@@ -219,6 +221,8 @@ func DisplayUsedConfig() {
 		log.Warn().Msg("could not load configuration file " + UserProvidedPath)
 	} else if LoadedConfig {
 		log.Info().Msg("loaded configuration from " + viper.ConfigFileUsed() + " using source " + Source)
+	} else if Source == configSourceBase64 {
```

Comment:
should we also cover the `MONDOO_CONFIG_PATH` case here?

Reply:
That was covered by `} else if LoadedConfig {`.

But this PR has a bug. I have to fix it to have it covered there again. Will push another commit.

Reply:
Fixed, now this case looks as before:
```
export MONDOO_CONFIG_PATH=/tmp/creds.yaml
 ~  cnquery status 
→ loaded configuration from /tmp/creds.yaml using source $MONDOO_CONFIG_PATH
→ Platform:	ubuntu
...
```